Integrations/integration-ServiceNowPython.yml

```diff
@@ -0,0 +1,1728 @@
+                  'Ticket(val.ID==obj.ID)': get_ticket_context(res['result']),
+            }
+        }
+
+        return entry
+
+
+    def query(ticket_type, sysparm_limit, sysparm_query):
+        query_params = {}
+        query_params['sysparm_limit'] = sysparm_limit
+        if sysparm_query:
+            query_params['sysparm_query'] = sysparm_query
+
+        path = 'table/' + ticket_type
+
+        return send_request(path, 'get', params = query_params)
+
+
+    def upload_file_command():
+        ticket_type = get_table_name(demisto.args().get('ticket_type', None))
+        ticket_id = demisto.args()['id']
+        file_id = demisto.args()['file_id']
+        file_name = demisto.args().get('file_name',
+                                    demisto.dt(demisto.context(), "File(val.EntryID=='"+file_id+"').Name"))
+        file_name = file_name[0] if isinstance(file_name, list) else file_name
+
+        res = upload_file(ticket_id, file_id, file_name, ticket_type)
+
+        hr = {
+            'Filename': res['result']['file_name'],
+            'Download link': res['result']['download_link'],
+            'System ID': res['result']['sys_id']
+        }
+
+        context = {
+            'ID': ticket_id,
+            'File': {}
+        }
+        context['File']['Filename'] = res['result']['file_name']
+        context['File']['Link'] = res['result']['download_link']
+        context['File']['SystemID'] = res['result']['sys_id']
+
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': res,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('File Uploaded', hr),
+            'EntryContext': {
+                'Ticket(val.ID==obj.ID)' : context
+            }
+        }
+
+        return entry
+
+
+    def upload_file(ticket_id, file_id, file_name, ticket_type):
+        headers = {
+            'Accept': 'application/json'
+        }
+
+        body = {
+            'table_name': ticket_type,
+            'table_sys_id': ticket_id,
+            'file_name': file_name
+        }
+
+        path = 'attachment/upload'
+
+        return send_request(path, 'post', headers = headers, body = body, file = {'id': file_id, 'name': file_name})
+
+
+    def get_computer_command():
+        ticket_type = 'cmdb_ci_computer'
+        computer_name = demisto.args()['computerName']
+
+        res = get_computer(ticket_type, computer_name)
+
+        if not res or 'result' not in res:
+            return 'Cannot find computer'
+        elif isinstance(res['result'], list):
+            if len(res['result']) == 0:
+                return 'Cannot find computer'
+            computer = res['result'][0]
+        else:
+            computer = res['result']
+
+        if computer['u_code'] != computer_name:
+            return 'Computer not found'
+
+        hr = {
+            'Id': computer['sys_id'],
```

Comment:
should be ID

Reply:
backward compatibility, can add ID in addition